spacewalk/setup/spacewalk-setup.spec

```diff
@@ -206,26 +188,22 @@ install -Dd -m 0755 %{buildroot}%{_prefix}/share/salt-formulas/states
 install -Dd -m 0755 %{buildroot}%{_prefix}/share/salt-formulas/metadata
 
 %post
+if [ -e /etc/tomcat/server.xml ]; then
+    cp /etc/tomcat/server.xml /etc/tomcat/server.xml.post-script-backup
+    xsltproc %{_datadir}/spacewalk/setup/server.xml.xsl /etc/tomcat/server.xml.post-script-backup > /etc/tomcat/server.xml
+fi
+
 if [ $1 == 2 -a -e /etc/tomcat/server.xml ]; then
 #during upgrade, setup new connectionTimeout if the user didn't change it
     cp /etc/tomcat/server.xml /etc/tomcat/server.xml.post-script-backup
     xsltproc %{_datadir}/spacewalk/setup/server_update.xml.xsl /etc/tomcat/server.xml.post-script-backup > /etc/tomcat/server.xml
 fi
```

Comment:
Do we still need this "extra" change? Can it be included in the other change?

Reply:
Honestly I don't remember why do it in this way...I'll try to resume the issue and in case fix it

Reply:
changes are required until 4.2 is maintained. I created 2 xsl files, one is executed just during a new installation, another only during upgrade

Reply:
Removing this will stop tomcat from starting for default Java installations (Enterprise Linux). The Leap Java version specifically has java.annotation and com.sun.xml.bind included.

Let me see how the difference can be integrated into your changes.

Reply:
Readding the a modified variant with https://github.com/uyuni-project/uyuni/pull/6846